src/aclib/ac_utils/ac_utils.cpp

```diff
@@ -148,20 +151,15 @@ args_t ac_init_args( int ac, char* av[]){
             av++;
         }
 #ifdef USE_GDB
-        //     if( (size>=5) && (!strncmp( av[1], "--gdb", 5))){ //Enable GDB support
-        //       int port = 0;
-        //       if ( size > 6 )
-        //         {
-        //           port = atoi( av[1] + 6 );
-        //           if ( ( port > 1024 ) && gdbstub )
-        //             gdbstub->set_port( port );
-        //         }
-        //       if ( gdbstub )
-        //         gdbstub->enable();
-        //     }
-#endif /* USE_GDB */
+        else if ((size >= 7) &&
```

Comment:
I think this should be greater than 7, and not greater or equal, right? Otherwise, if the user types "--gdb-port=" without typing any number after this string, it will load an unknown port because I'm not sure what atoi returns if it is asked to convert "\0" to a number.